industrial_ci/src/builders/catkin_make.sh

```diff
@@ -33,15 +39,15 @@ function builder_run_build {
     local ws=$1; shift
     local opts=()
     _append_job_opts opts PARALLEL_BUILDS 0
-    ici_exec_in_workspace "$extend" "$ws" catkin_make --make-args install "${opts[@]}" "$@"
+    _run_catkin_make install "$extend" "$ws" "${opts[@]}" "$@"
 }
 
 function builder_run_tests {
     local extend=$1; shift
     local ws=$1; shift
-    local opts=()
+    local opts=(--force-cmake)
     _append_job_opts opts PARALLEL_TESTS 1
-    ici_exec_in_workspace "$extend" "$ws" catkin_make --make-args run_tests "${opts[@]}" "$@"
+    _run_catkin_make run_tests "$ws/devel" "$ws" "${opts[@]}" "$@"
```

Comment:
Was this broken already before all these changes? How did the tests find there dependencies from the `$extend` upstream only?
Out of curiosity: Why don't you use `$(ici_extend_space "$ws")` here as well - as below for `catkin_tools`?

Reply:
> Was this broken already before all these changes? How did the tests find there dependencies from the $extend upstream only?

Perhaps. I am not sure, if the source space was added during the tests somehow.
It might be that we do not have to source the target space at all.

> Out of curiosity: Why don't you use $(ici_extend_space "$ws") here as well - as below for catkin_tools?

That's what `ros_buildfarm` is doing (https://github.com/ros-industrial/industrial_ci/pull/704#issuecomment-881493215).
`$(ici_extend_space "$ws")` might be better, I will give it a try.